cd/manager/jobs/smoke.go

```diff
@@ -9,33 +9,72 @@ import (
 	"github.com/3box/pipeline-tools/cd/manager"
 )
 
+// Allow up to 1 hours for smoke tests to run
+const SmokeTestFailureTime = 1 * time.Hour
```

Comment:
Is this too long? Lambda test timeout was 5 minutes, so maybe 10-15 min max here? WDYT?

Reply:
will set to 15 min